fhir-bulkdata-webapp/src/main/java/com/ibm/fhir/bulkdata/export/patient/resource/PatientResourceHandler.java

```diff
@@ -125,7 +125,7 @@ public List<Resource> executeSearch(Set<String> patientIds) throws Exception {
 
             do {
                 searchContext.setPageNumber(compartmentPageNum);
-                FHIRPersistenceContext persistenceContext = FHIRPersistenceContextFactory.createPersistenceContext(null, searchContext);
+                FHIRPersistenceContext persistenceContext = FHIRPersistenceContextFactory.createPersistenceContext(null, searchContext, null);
```

Comment:
makes me wonder whether/how to support bulk export from the distributed citus backend...